sound/soc/sof/intel/hda-loader.c

```diff
@@ -206,18 +206,15 @@ static int cl_trigger(struct snd_sof_dev *sdev,
 }
 
 static struct hdac_ext_stream *get_stream_with_tag(struct snd_sof_dev *sdev,
-						   int tag)
+						   int tag, int direction)
```

Comment:
typo in commit message, should be ASoC: SOF: Intel.
Maybe fix in sof-dev-rebase to avoid unnecessary reviews?